docs/dto-structure.md

```diff
@@ -0,0 +1,39 @@
+Fazland - DtoManagementBundle - DTO structure
+=============================================
+DtoManagementBundle has an opiniated way to implement DTOs:
+- Choose a namespace (or multiple namespaces) that will contain your DTOs
+- Inside the namespaces create subnamespace containing the interfaces of the DTOs and another one containing all the implementations.
+
+Example:
+`App\Model` is the base namespace configured
+`App\Model\Interfaces` is the subnamespace containing all the interfaces
+`App\Model\v2018` is the subnamespace containing all 2018th implementations.
+
+Why 2018? Because we structure the DTOs' versions based on release date in production of the DTOs.
```

Comment:
Currently, this is a limitation of the bundle that will not accept any other version format.
The limitation is [here](https://github.com/fazland/dto-management-bundle/blob/master/src/DependencyInjection/DtoManagementExtension.php#L126) and probably in a compiler pass because of this.

I think that we should recommend this structure, document that this is the only way for now because is a known limitation, but we are open to another implementation that is not limited to this.

Reply:
Yep, that was something I wanted to explain better